README.md

```diff
@@ -1,112 +1,62 @@
+with Kubeflow Training to orchestrate their ML training on Kubernetes.
 
-### Installing the Python SDK
+Kubeflow Trainer allows you effortlessly develop your LLMs with the Kubeflow Python SDK and
+build Kubernetes-native Training Runtimes with Kubernetes Custom Resources APIs.
 
-The Training Operator [implements a Python SDK](https://pypi.org/project/kubeflow-training/)
-to simplify creation of distributed training and fine-tuning jobs for Data Scientists.
+## Kubeflow Trainer Introduction
 
-Run the following command to install the latest stable release of the Training SDK:
+The following KubeCon + CloudNativeCon 2024 talk provides an overview of Kubeflow Trainer capabilities:
 
-```
-pip install -U kubeflow-training
-```
+[![Kubeflow Trainer](https://img.youtube.com/vi/Lgy4ir1AhYw/0.jpg)](https://www.youtube.com/watch?v=Lgy4ir1AhYw)
 
 ## Getting Started
 
-Please refer to [the getting started guide](https://www.kubeflow.org/docs/components/training/getting-started/#getting-started-with-pytorchjob)
-to quickly create your first distributed training job using the Python SDK.
-
-If you want to work directly with Kubernetes Custom Resources provided by Training Operator,
-follow [the PyTorchJob MNIST guide](https://www.kubeflow.org/docs/components/training/pytorch/#creating-a-pytorch-training-job).
+Please check [the official Kubeflow documentation](https://www.kubeflow.org/docs/components/training/getting-started)
+to install and get started with Kubeflow Trainer.
 
 ## Community
 
 The following links provide information on how to get involved in the community:
 
-- Attend [the bi-weekly AutoML and Training Working Group](https://bit.ly/2PWVCkV) community meeting.
 - Join our [`#kubeflow-training` Slack channel](https://www.kubeflow.org/docs/about/community/#kubeflow-slack).
+- Attend [the bi-weekly AutoML and Training Working Group](https://bit.ly/2PWVCkV) community meeting.
 - Check out [who is using the Training Operator](ADOPTERS.md).
 
-This is a part of Kubeflow, so please see [readme in kubeflow/kubeflow](https://github.com/kubeflow/kubeflow#get-involved) to get in touch with the community.
-
 ## Contributing
 
 Please refer to the [CONTRIBUTING guide](CONTRIBUTING.md).
 
-## Change Log
+## Changelog
 
 Please refer to the [CHANGELOG](CHANGELOG.md).
 
-## Version Matrix
-
-The following table lists the most recent few versions of the operator.
-
-| Operator Version       | API Version | Kubernetes Version |
-| ---------------------- | ----------- | ------------------ |
-| `v1.4.x`               | `v1`        | 1.23+              |
-| `v1.5.x`               | `v1`        | 1.23+              |
-| `v1.6.x`               | `v1`        | 1.23+              |
-| `v1.7.x`               | `v1`        | 1.25+              |
-| `v1.8.x`               | `v1`        | 1.27+              |
-| `latest` (master HEAD) | `v1`        | 1.27+              |
-
-## Reference
+## Kubeflow Training Operator V1
```

Comment:
We should work on this message to explain that users can still use Training Operator v1 and we will resolve any bugs on the release branch. cc @kubeflow/wg-training-leads